sds_data_manager/lambda_code/IAlirtCode/ialirt_ingest.py

```diff
@@ -255,11 +260,21 @@ def process_algorithms(combined: xr.Dataset, algorithm_table):
             download_path = get_ancillary(instrument, "l1b-calibration")
             calibration_data = load_cdf(download_path)
             result, _ = process_func(combined, calibration_data)
+        elif instrument == "codicelo":
+            result, _ = process_func(combined)
+        elif instrument == "codicehi":
+            _, result = process_func(combined)
```

Comment:
Just wanted to note here that this is something I would like to revisit after the SIT. I think the way my code is set up right now isn't very efficient (since both `codicelo` and `codicehi` is processed regardless of which instrument is needed). But this is good enough for now I think.